core/src/main/java/tech/pegasys/web3signer/core/service/http/ArtifactType.java

```diff
@@ -23,5 +23,6 @@ public enum ArtifactType {
   VOLUNTARY_EXIT,
   SYNC_COMMITTEE_MESSAGE,
   SYNC_COMMITTEE_SELECTION_PROOF,
-  SYNC_COMMITTEE_CONTRIBUTION_AND_PROOF
+  SYNC_COMMITTEE_CONTRIBUTION_AND_PROOF,
+  REGISTER_VALIDATOR_MESSAGE
```

Comment:
Unsure of the name, the type in Teku is ValidatorRegistration. Would VALIDATOR_REGISTRATION or VALIDATOR_REGISTRATION_MESSAGE be more correct?

Reply:
Maybe register validator would be better? since the request consists both a validator registration and an epoch.

Reply:
🤷 Not sure. That's inconsistent with the name used in Teku. Would rather it was consistent everywhere.

Reply:
Looking at the spec, the message type is ValidatorRegistration so would rather is was consistent with that. We do add the epoch as well but don't think that matters.

Reply:
Naming updated